### ISSUE TITLE
Better contrast for FAB

### ISSUE THREAD

**Issue by claudiofus** [mtotschnig/MyExpenses#697]
I suggest a better contrast for FAB, expecially with dark colors (red, blue, grey) it's difficult to see the + symbol in the FAB.
You can also consider [Contrast Ratio](https://contrast-ratio.com/) in order to offer a better UX and respect WCAG 2.0.

**Comment by mtotschnig** [comment]
Currently the app dynamically sets the icon color to black or white based on which of them gives a higher contrast with the fab color. This might not work on API level <21. Could you tell on which Android version you were making your observation? Possibly also post that part of a screenshot showing how the FAB is rendered?

**Comment by claudiofus** [comment]
I'm using android 10, here you can see the rendering. PS: I have installed the latest version.
![20201104_225101](https://user-images.githubusercontent.com/18471299/98171761-8bb7e080-1ef0-11eb-9442-1f4cdc00be0b.jpg)

**Comment by mtotschnig** [comment]
What MyExpenses does, it compares the contrast ratio between the given account color and black or white, and then selects either of them. In your case, the color seems to be #009ACB.
According to https://webaim.org/resources/contrastchecker/ black gives a contrast of 6.48:1 and white 3.23:1. Thus it picks black.

**Comment by claudiofus** [comment]
I understand, it was just my opinion about it. Feel free to close that issue!
